### PR TITLE
ZH-trans: add hello-minikube.md

### PR DESCRIPTION
**Pull request by xichengliudui**

### CODE REVIEW

File: content/zh/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+{{% capture prerequisites %}}
+
+<!--
+This tutorial provides a container image built from the following files:
+-->
+本教程提供了一个由以下文件构建的容器镜像：
+
+{{< codenew language="js" file="minikube/server.js" >}}
+
+{{< codenew language="conf" file="minikube/Dockerfile" >}}
```
Comment (tengqm) [inline content/zh/docs/tutorials/hello-minikube.md]: The above two files need to be copied.

Comment (xichengliudui) [reply content/zh/docs/tutorials/hello-minikube.md]: thanks.

**Comment by markthink** [comment]
@xichengliudui  编译有问题哦

**Comment by tengqm** [comment]
@xichengliudui This one has been there for quite a long time. Still working on it?

**Comment by xichengliudui** [comment]
The current work is based on 1.14. What I want is to close this pr and create a new pr based on 1.14. Is that appropriate?

**Comment by tengqm** [comment]
Yes. Please.

已使用 IBM Verse 从我的 iPhone 发送

在 2019年8月6日，13:26:26，notifications@github.com 写道：

**Comment by xichengliudui** [comment]
@tengqm , thanks, i found this article has been translated!

**Pull request closed without merging**